.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 node_js:
   - stable
+  - 6
 cache: yarn
+install: yarn --ignore-engines # ignore engines to test node 6, otherwise it fails on engine check
```

Comment:
Instead of this, we should probably change the requirement in package.json to require only node 6

Reply:
Actually you've done that... so we can remove the `--ignore-engines` flag, yes?

Reply:
I added it because flow-copy-source has node engine>=8.0
However, with the flag it doesn't break anything, at least for now.
If toolchain has higher requirement later, we may try building modules with node 8, then run tests in node 6. Configuration is complicated, so I think, before it actually breaks we can just keep this flag and build && test in node 6.